sdk/cloudmachine/README.md

```diff
@@ -1,143 +1,93 @@
 # Azure Projects client library for .NET
 
-Write Azure apps in 5 minutes
+Azure.Projects is an set of libraries and tools for rapid application development on Azure.
+If you cannot get an app running in 10 minutes, let us know that we failed!
 
-## Getting started
-
-### Install the packages
+These libraries and tools make it easy to get apps off the ground, by:
```

Comment:
```suggestion
These libraries and tools make it easier to start building an application, by:
```

MS Docs guidelines suggest avoiding words like "easy" because it is subjective and may cause people who don't feel this is easy to feel negatively.

Reply:
I have to say I am not thrilled about this suggestion. It reads like hedging bets.

Reply:
What's easy for Person A may be difficult for Person B.    If Person B struggles with this, the tone of the guide implies "this should be easy.  You're not good at this and should feel bad."

I'll look for the official guideline doc that discusses when I get a moment, but @scottaddie may know where it is off the top of his head.

Reply:
Conversely, you could frame as aspirational and remove the connotation.  Something like "The goal of these libraries is to make it easy to...."